.circleci/config.yml

```diff
@@ -1111,6 +1141,63 @@ workflows:
       - node-tedious
       - node-when
       - node-winston
+      - codecov:
+          requires:
+            - node-core-8
```

Comment:
This whole list is repeated 3 or 4 times in this file. Can DRY it up?

Reply:
Doesn't look like it unfortunately as the idea was [rejected](https://github.com/yaml/yaml/issues/35).